Source/WebCore/page/Navigation.cpp

```diff
@@ -140,6 +138,14 @@ void Navigation::initializeForNewWindow(std::optional<NavigationNavigationType>
                 updateForActivation(currentItem.get(), navigationType);
                 return;
             }
+            if (navigationType == NavigationNavigationType::Traverse) {
+                m_currentEntryIndex = getEntryIndexOfHistoryItem(m_entries, *currentItem);
+                if (m_currentEntryIndex) {
```

Comment:
Do you remember which test case a traversal happened but wasn't in the entries? I would think that is never allowed.

Reply:
I think it was navigation-activation/activation-replace.html, or otherwise one of the other navigation-activation tests.